modules/appeals_api/config/schemas/shared/v0/address.json

```diff
@@ -21,22 +21,24 @@
             "WI", "WY"
           ]
         },
-        "countryCodeISO2": {
+        "countryCodeIso3": {
+          "description": "Three letter ISO 3166-1 country code",
           "type": "string",
-          "pattern": "^[A-Z]{2}$",
-          "minLength": 2,
-          "maxLength": 2
+          "pattern": "^[A-Z]{3}$",
+          "minLength": 3,
+          "maxLength": 3,
+          "example": "USA"
         },
         "zipCode5": {
           "type": "string",
-          "description": "5-digit zipcode. Use \"00000\" if address is outside the United States",
```

Comment:
I noticed in our Lighthouse standards that fields with a number of digits/letters would use spelled out single numerical values and a space instead of a dash, so I updated both instances here to use the same style